scripts/docker/dev/start-django.sh

```diff
@@ -21,6 +21,13 @@ fi
 npm run devw&
 popd
 
+pushd akvo/rsr/spa
+if [[ ! -d "node_modules" ]]; then
```

Comment:
Maybe there is a better way to know if `npm install` must be run?

Reply:
Not sure what you mean, but `npm install` does it's own check of whether there's anything missing in "node_modules" and installs only then.

Reply:
Checking if the directory `node_modules` exists before running `npm install` is probably due to `npm install` is/was too slow to run in each and every restart of the container. 

Maybe this is no longer the case, or the delay is acceptable, or the npm install can be run in the background the same way that `npm start` is run in line 27?

Reply:
`npm install` does take a while because it also checks for updates to the packages. It might actually be good if we can choose when to run an 'npm install' by deleting the node_modules folder when publishing.

Reply:
Actually I just checked `yarn install` vs `npm install` and the result is 1 second vs 15-60 seconds.

Reply:
There is the option to use docker layering to run `npm install` just when the package.json has changed, but that means not being able to see the node_modules content from the host machine (so not being able to find see it in your Mac filesystem, which probably means not being able to browse the source code of the dependencies from your IDE).

Or we could implement our own change detection mechanism of the package.json.

Reply:
In this case I think it would be better if we just use yarn and run `yarn install` every time.

Reply:
Agree. Didn't see your `yarn` comment before posting my comment.